WooCommerce/Classes/POS/Presentation/CartView.swift

```diff
@@ -102,7 +103,7 @@ struct CartView: View {
 
 private extension CartView {
     enum Constants {
-        static let primaryFont: Font = .system(size: 40, weight: .bold, design: .default)
+        static let primaryFont: Font = Font.posTitle
         static let secondaryFont: Font = .system(size: 24, weight: .regular, design: .default)
         static let itemsFont: Font = .system(size: 16, weight: .medium, design: .default)
         static let clearButtonFont: Font = .system(size: 16, weight: .semibold, design: .default)
```

Comment:
I felt these outside of this PR just to narrow down testing, happy to change them to also use the POS defined fonts if needed 👍

Reply:
Nice! Done on https://github.com/woocommerce/woocommerce-ios/pull/13631/commits/66a57bb2381d415462b440e0e624cc127afe7ed5